### ISSUE TITLE
Posting links do not work

### ISSUE THREAD

**Issue by athosmoreno** [FireCubeStudios/DarkSky#38]
When posting a link via DarkSky, it is displayed as clear text and cannot be clicked. Links posted by others can be clicked.

Example:

![image](https://github.com/user-attachments/assets/86e94b28-a9c7-4a0d-917e-cf3ddd305183)
![image](https://github.com/user-attachments/assets/fb872460-7729-4b63-814d-c46854ec3f80)

**Comment by FireCubeStudios** [comment]
Yes this is a known issue, I have only added basic text posts creation so far. #12 tracks more advanced features
